permhash/helpers.py

```diff
@@ -179,17 +206,26 @@ def create_crx_permlist(path):
                 path,
             )
             return False
-        try:
-            manifest_json = json.loads(manifest_text)
-        except JSONDecodeError:
+        if "//" in manifest_text or "/*" in manifest_text:
+            manifest_json = strip_comments(manifest_text, path)
+        else:
             try:
-                manifest_json = json.loads(manifest_text.encode().decode("utf-8-sig"))
-            except JSONDecodeError:
-                logging.warning(
-                    "This Manifest file is abnormal and unable to be read: %s.", path
-                )
+                manifest_json = json.loads(manifest_text)
+            except OSError:
+                logging.warning("This manifest is unable to be read: %s.", path)
                 return False
-        perm_list = parse_crx_manifest(manifest_json)
+            except JSONDecodeError:
+                try:
+                    manifest_json = json.loads(
+                        manifest_text.encode().decode("utf-8-sig")
+                    )
+                except JSONDecodeError:
+                    logging.warning(
+                        "This manifest file is abnormal and unable to be read: %s.",
+                        path,
+                    )
+                    return False
+        perm_list = parse_crx_manifest(manifest_json, path)
```

Comment:
Related to the comment on path, I'd recommend adding a check here to see if perm_list is False here and log that there was an error

Reply:
That's exactly how it was handled with moving the path. We check if it is false and raise as needed.